src/test/scala/firrtl/backends/experimental/smt/FirrtlModuleToTransitionSystemSpec.scala

```diff
@@ -136,108 +137,6 @@ private class FirrtlModuleToTransitionSystemSpec extends SMTBackendBaseSpec {
     assert(false, "TODO")
   }
 
-  it should "support memories with registered read port" in {
```

Comment:
I removed these two tests because they were to brittle and the functionality is already tested as part of the end2end tests.